### PR TITLE
Lets medical belts hold more types of stuff

### PR DESCRIPTION
**Pull request by Bawhoppen**

### CODE REVIEW

File: code/game/objects/items/weapons/storage/belt.dm [hunk 1]
```diff
@@ -80,7 +81,29 @@
...
+		/obj/item/clothing/mask/breath,
+		/obj/item/clothing/mask/breath/medical,
+		/obj/item/weapon/surgical_drapes, //for true paramedics
+		/obj/item/weapon/scalpel,
+		/obj/item/weapon/circular_saw,
+		/obj/item/weapon/surgicaldrill,
+		/obj/item/weapon/retractor,
+		/obj/item/weapon/cautery,
+		/obj/item/weapon/hemostat,
+		/obj/item/device/geiger_counter,
```
Comment (PKPenguin321) [inline code/game/objects/items/weapons/storage/belt.dm]: One of these things is not like the others

**Comment by PKPenguin321** [comment]
This is pretty much fine imo

**Comment by Shapsy** [comment]
>/obj/item/weapon/bikehorn/rubberducky
>/obj/item/weapon/circular_saw
>/obj/item/weapon/surgicaldrill

but why these

File: code/game/objects/items/weapons/storage/belt.dm (hunk 1 above)
Comment (Bawhoppen) [inline code/game/objects/items/weapons/storage/belt.dm]: I plead to the fifth.

**Comment by Bawhoppen** [comment]
all surgery tools

**Comment by PKPenguin321** [comment]
>15 brute weapons that can dismember shouldn't be able to slot into a belt

Yeah but they can already fit in your backpack anyways so honestly it's not really a big deal

**Comment by Bawhoppen** [comment]
why not the duck

**Comment by Shapsy** [comment]
This would let you carry 7 normal sized saws/drills in 1 normal sized belt

**Comment by Bawhoppen** [comment]
give it a rest shaps

**Comment by PKPenguin321** [comment]
Where are you getting 7 drills as a doctor and also why

**Comment by Bawhoppen** [comment]
also you know that saws never break so you only ever need 1, 2 maybe if you get slipped and drop it

**Comment by KorPhaeron** [comment]
Bawhoppen will never rest until belts are just second backpacks

**Comment by Niknakflak** [comment]
Why the lazarus injector?

**Comment by Jalleo** [comment]
instant revival of mobs like pun pun.

**Comment by Archie700** [comment]
The mental image of a doctor with a circular saw holstered in his belt is too funny.

Other than that, it's ok for doctors who want to be paramedics.

**Comment by Bawhoppen** [comment]
@Niknakflak lazarus injector is very medical, 
also for animal hospital it makes sense

**Comment by AdamElTablawy** [comment]
i don't see an issue because you can just put drills or saws in your backpack for the same effect and there's no reason to have more than one saw in the belt

**Comment by Jalleo** [comment]
I dont think you can even easily get a saw if cargo/R&D refuse you.

File: code/game/objects/items/weapons/storage/belt.dm (hunk 1 above)
Comment (Xhuis) [inline code/game/objects/items/weapons/storage/belt.dm]: It's ghetto, but it makes sense for checking exact radiation levels.

**Comment by KorPhaeron** [comment]
I dont think encouraging people to steal all the medical tools is good

**Comment by Endozworld** [comment]
@KorPhaeron I agree, it's bad enough having the saw disappear with the new limb detachment, cant imagine losing anything else

**Pull request merged**